pages/learn/develop/hardware/i2c-and-spi.md

```diff
@@ -234,6 +234,14 @@ cat /sys/devices/platform/ocp/ocp:cape-universal/status
 OCPDIR=/sys/devices/platform/ocp/ocp*
 SLOTS=/sys/devices/platform/bone_capemgr/slots
 ```
+### Adding an Overlay
+
+To load an overlay in more recent versions of balenaOS, it's necessary to edit the boot partition uEnv.txt file (or uEnv.txt_internal on the flasher image) before provisioning and add the following content:
```

Comment:
@kenna-smith if you write here provisioning it's implied that it uses a flasher image. So the reference would be actually only to uEnv.txt_internal